src/StarWars/StarWarsSchema.cs

```diff
@@ -9,8 +9,10 @@ public class StarWarsSchema : Schema
         public StarWarsSchema(IServiceProvider provider)
             : base(provider)
         {
-            Query = provider.GetRequiredService<StarWarsQuery>();
-            Mutation = provider.GetRequiredService<StarWarsMutation>();
+            Query = (StarWarsQuery)provider.GetService(typeof(StarWarsQuery)) ?? throw new InvalidOperationException();
+            Mutation = (StarWarsMutation)provider.GetService(typeof(StarWarsMutation)) ?? throw new InvalidOperationException();
```

Comment:
Why?

Reply:
Long time ago we removed `GetRequiredService` extension method from GraphQL.NET codebase.